src/test/java/TestReadTextFromFile.java

```diff
@@ -0,0 +1,31 @@
+
+import org.junit.Assert;
+import org.junit.Test;
+import java.io.IOException;
+
+public class TestReadTextFromFile {
+
+    @Test
+    public void reedingTextFromFile() throws IOException {
+
+        ReadTextFromFile readTextFromFile = new ReadTextFromFile();
+        String textFromFile =" ".concat("Shortly before the 2020 election government-sponsored program " +
+                "$3 billion for");
+        String pathnameInputFile = "src/main/resources/books/InputTest";
+        boolean bool = textFromFile.equals(readTextFromFile.reedingTextFromFile(pathnameInputFile));
+        Assert.assertArrayEquals(new boolean[]{true},new boolean[]{bool});
```

Comment:
it looks like you don't need to wrap boolean  true and variable bool in to the array, just compare them directly